app/controllers/products_controller.rb

```diff
@@ -8,7 +8,7 @@ class ProductsController < ApplicationController
   add_breadcrumb "Products", :shop_products_path
 
   def index
-    @products = Product.all
+    @products = Product.all.page params[:page]
```

Comment:
You have added to the products view here, but haven't included it in the view code?